addons/html_builder/static/src/builder/builder_components/utils.js

```diff
@@ -522,15 +573,16 @@ export function getAllActionsAndOperations(comp) {
         if (actionId) {
             actions.push({ actionId, actionParam, actionValue });
         }
-        const inheritedActions = comp.props.inheritedActions
-            ?.map(
-                (actionId) =>
-                    comp.env.dependencyManager
-                        // The dependency might not be loaded yet.
-                        .get(actionId)
-                        ?.getActions?.() || []
-            )
-            .flat();
+        const inheritedActions =
+            inheritedActionIds
+                ?.map(
+                    (actionId) =>
+                        comp.env.dependencyManager
+                            // The dependency might not be loaded yet.
+                            .get(actionId)
+                            ?.getActions?.() || []
```

Comment:
missing getActions ?

Reply:
dependency without getActions

Reply:
a dependency could be a "selectable" or a "clickable"